### PR TITLE
add flexible size

### PR DESCRIPTION
**Pull request by hugo-clemente**
Hey,
First of all thanks for the work, just wanted to add the `flexible` size, but had to update the `turnstile-types`. 
I read in the previous PR that you wanted to preserve the `invisible` option, but that makes the types inconsistent between `react-turnstile` and `turnstile-types` so i chose to remove it.

### CODE REVIEW

**Comment by Le0Developer** [comment]
`expired-callback` does actually get the current token as a parameter. Must've forgotten it in `turnstile-types`, will push an update to it soon™️. See below from api.js:
image

Also your style override for width/height is in conflict/duplicated by the `fixedSize` prop.

**Comment by Le0Developer** [comment]
I've released v1.2.3 of turnstile-types which adds `invisible` to `size` and `token` to `expired-callback`

File: src/index.tsx [hunk 1]
```diff
@@ -208,8 +208,6 @@ export default function Turnstile({
         fixedSize
           ? {
               ...(style ?? {}),
-              width: size === "compact" ? "130px" : "300px",
-              height: size === "compact" ? "120px" : "65px",
```
Comment (Le0Developer) [inline src/index.tsx:206]: so `fixedSize` now does nothing? 🤨

**Comment by hugo-clemente** [comment]
Yeah sorry had a few back and forth with myself... 😅 
It should be good now though, used the updated version of `turnstile-types`, and fixed the issue with the `fixedSized` prop

**Comment by Le0Developer** [comment]
Last missing piece is an entry in HISTORY.md 🙏

**Comment by hugo-clemente** [comment]
Done !

**Review by Le0Developer — APPROVED** [review]
I'm puzzled by the CI failure, it's fine locally.
I suspect it might be because the CI is running node 18 and thus runs an older prettier version that still supports it, but I'll fix that in a separate PR.

**Pull request merged**